tests/providers/test_qwen_asr_session.py

```
@@ -193,14 +469,22 @@ def append_audio(self, audio_b64: str):
+            self.callback.on_event(
+                {"type": "input_audio_buffer.committed", "item_id": item_id}
+            )
             self.callback.on_event(
                 {
                     "type": "conversation.item.input_audio_transcription.completed",
+                    "item_id": item_id,
                     "transcript": "final transcript",
                 }
             )
 
+        def end_session(self, timeout):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `end_session`

Add return type annotation: `None`

(ANN202)